Source/Csla.Shared/DataPortal.cs

```diff
@@ -733,7 +733,10 @@ internal static void LoadDataPortalProxyFactory()
         }
         else
         {
-          var proxyFactoryType = Type.GetType(ApplicationContext.DataPortalProxyFactory);
+          var proxyFactoryType = 
+            Type.GetType(ApplicationContext.DataPortalProxyFactory) ?? 
+            throw new InvalidOperationException($"Unable to load DataPortalProxyFactory {ApplicationContext.DataPortalProxyFactory}");
```

Comment:
Exception text needs to come from a resource file, otherwise it can't be localized into other languages. 

I too love the new $"" capabilities, but the resource strings all use $0, $1, etc. type parameters and we should keep that consistent as well.

Reply:
Ahh, sorry, I forgot about the localization requirements.  I'll make the changes needed.